thoth-app/src/component/issues_form.rs

```diff
@@ -477,14 +477,14 @@ impl IssuesFormComponent {
                     <div class="field" style="width: 8em;">
                         <label class="label">{ "ISSN Print" }</label>
                         <div class="control is-expanded">
-                            {&i.series.issn_print}
+                            {&i.series.issn_print.clone().unwrap_or_default()}
```

Comment:
you can just use the refefence to avoid cloning `&i.series.issn_print.as_ref().unwrap_or(&String::default())`